tasks/cea-install.js

```diff
@@ -1,30 +1,82 @@
 #!/usr/bin/env node
 /* eslint no-console: 0 */
 
-const cp = require('child_process');
+const execSync = require('child_process').execSync;
+const spawn = require('cross-spawn');
+const path = require('path');
 
-const TARGET_DIR = process.argv[2]; 
-const CLEANUP_DIRS = ['docs', '.circleci', '.github', '.git', '.npmignore', 'tasks'];
+let TARGET_DIR;
+const CLEANUP_DIRS = [
+  'docs',
+  '.circleci',
+  '.github',
+  '.git',
+  '.npmignore',
+  'tasks'
+];
 
 console.log('-------------------------------------------------------');
 console.log('Welcome to Create Evergreen App ♻️');
 console.log('-------------------------------------------------------');
 
-// overwrite existing directory or not?
-cp.execSync(`rm -rf ${TARGET_DIR}`);
+const checkTargetDir = appDir => {
+  if (appDir) {
+    TARGET_DIR = appDir;
+  } else {
+    console.error(
+      'Missing Project Directory! Please specifiy the application name e.g. create-evergreen-app my-app'
+    );
+    process.exit(1);
```

Comment:
The script previously also did process.exit but you included:

```
// eslint-disable-line no-process-exit
```

I assume throws error won't necessarily cause it to exit, which is what we want. Instead, probably the more correct method:

```
Process.exit();
var exit = process.exit;
```

Reply:
You're right throw new error will close only because of the try/catch statement preceeding the process.exit at the bottom(which is ignored by lint). In that case, we get an output:

```
node ./tasks/cea-install.js
-------------------------------------------------------
Welcome to Create Evergreen App ♻️
-------------------------------------------------------
Error: Missing Project Directory! Please specifiy the application name e.g. create-evergreen-app my-app
    at checkTargetDir (/workspace/evergreen/temp/create-evergreen-app/tasks/cea-install.js:26:11)
    at run (/workspace/evergreen/temp/create-evergreen-app/tasks/cea-install.js:53:5)
    at Object.<anonymous> (/workspace/evergreen/temp/create-evergreen-app/tasks/cea-install.js:81:1)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
    at startup (internal/bootstrap/node.js:266:19)
```

Which isn't ideal.

Reply:
Ok, let's leave it with the current implementation and the eslint `ignore` rule, since the output is a bit cleaner for users that way.  👍